crabpy/gateway/capakey.py

```diff
@@ -491,6 +491,50 @@ def creator():
         perceel.set_gateway(self)
         return perceel
 
+    def get_perceel_by_coordinates(self, x, y):
+        '''
+        Get a `perceel`.
+
+        :param capakey: An capakey for a `perceel`.
+        :rtype: :class:`Perceel`
+        '''
+
+        def creator():
+            url = self.base_url + '/parcel?x=%s&y=%s' % (x, y)
+            h = self.base_headers
+            p = {
+                'geometry': 'full',
+                'srs': '31370',
+                'data': 'adp'
```

Comment:
Kan je hier al een 'status': 'actual' aan toevoegen. We gaan binnenkort switchen naar v2 van de capakey service en willen vanaf dan de actuele perceelsinformatie binnenhalen.

Reply:
Goed. Er is nu wel veel overlap met de "get_perceel_by_capakey" functie, enkel de url is anders.
Misschien probeer ik best nog die gedeelde functionaliteit in een apparte functie te steken.